tools/topology/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ set(TPLGS
 	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-cml-rt711-rt1308-mono-rt715\;-DPLATFORM=cml-mono"
 	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-cml-rt711-rt1308-rt715\;-DPLATFORM=cml"
 	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-icl-rt711-rt1308-rt715-hdmi\;-DPLATFORM=icl"
+	"sof-icl-da7219\;sof-icl-da7219\;-DPLATFORM=icl"
```

Comment:
If you would like to use PLATFORM variable here, you need to make modify in m4 also.
Please also generate JSL tplg file here.

Reply:
Ack.

Reply:
Thanks for the code review, the topology is targeted for jsl, think it's better to make it standalone for jsl right now, and add support for other platforms only when needed to reduce the validation workload.